conda/meta.yaml

```diff
@@ -9,10 +9,6 @@ source:
 build:
   noarch: generic
   script: python -m pip install --no-deps --ignore-installed .
```

Comment:
Can you replace `python` here with ``{{ PYTHON }}``? This will make sure it uses the correct python when installing the package during the build step.